.lintstagedrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can we switch to `lint-staged.config.js` as the default config for `lint-staged`. The JS style gives us more flexibility.

Reply:
I'd rather not. But it's only my personal preference to introduce more flexibility when needed. Feel free to incorporate the changes from this PR into #5462.